src/components/video/Video.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import { StyledFigure, VideoContainer, PlaceholderButton } from './Video.styles'
+import { VideoProps } from './Video.types'
+
+const Video = ({ videoId, coverImage, captionString }: VideoProps) => {
+  const [clicked, setClicked] = useState(false)
+
+  return (
+    <StyledFigure>
+      {clicked ? (
+        <VideoContainer>
+          <iframe
+            allowFullScreen
+            frameBorder="0"
+            src={`https://www.youtube.com/embed/${videoId}?autoplay=1`}
+            title="Youtube video"
+          />
+        </VideoContainer>
+      ) : (
+        <div onClick={() => setClicked(!clicked)}>
+          <img src={coverImage} alt={captionString} />
```

Comment:
Can we use Gatsby's image here? It is much better for performance and responsiveness 🙂